### ISSUE TITLE
`check-db.js` script should report the actual cause of a connection error

### ISSUE THREAD

**Issue by dpk** [umami-software/umami#2837]
### Describe the feature or enhancement

While trying to install Umami I was stuck for quite a while at this stage:

```
✓ DATABASE_URL is defined.
✗ Unable to connect to the database.
```

The `check-db.js` script does not report what the actual exception raised by Prisma was. Credentials were correct, I was able to do a manual connection with `psql` with the given password etc., but this error persisted.

The problem turned out to be that Prisma was not finding the right version of the OpenSSL library*. But finding this out involved quite a bit of guesswork and wasted over an hour of my life.

The `check-db.js` script should actually show the exception that caused the connection to fail.

(* Why on earth it needs this in order to connect to a local database *without* SSL is anyone’s guess …)

**Comment by mikecao** [comment]
It should report it now in the latest version.

**Comment by boly38** [comment]
Hi, I'm using [provided docker compose](https://github.com/umami-software/umami/blob/master/docker-compose.yml) and I'm having this issue too.

I would suggest getting more details about this error too (if possible).


NB: if needed umami could add an optional ENV variable to switch at DEBUG or VERBOSE level if it helps to solve some configuration issues (without impacting the current legacy startup logs level/verbosity)

**Comment by IvanGnusarkov** [comment]
@dpk, i fixed it in #2786, released in v2.13.0
